controls/pcidss_4_ocp4.yml

```diff
@@ -0,0 +1,3591 @@
+      status: pending
+
+  - id: '12.10'
+    title: Suspected and confirmed security incidents that could impact the CDE are responded to
+      immediately.
+    levels:
+      - base
+    status: pending
+    controls:
+    - id: 12.10.1
+      title: An incident response plan exists and is ready to be activated in the event of a
+        suspected or confirmed security incident.
+      levels:
+        - base
+      status: pending
+
+    - id: 12.10.2
+      title: At least once every 12 months, the security incident response plan is reviewed,
+        updated, and tested.
+      levels:
+        - base
+      status: pending
+
+    - id: 12.10.3
+      title: Specific personnel are designated to be available on a 24/7 basis to respond to
+        suspected or confirmed security incidents.
+      description: |-
+        Incidents are responded to immediately where appropriate.
+      levels:
+        - base
+      status: pending
+
+    - id: 12.10.4
+      title: Personnel responsible for responding to suspected and confirmed security incidents
+        are appropriately and periodically trained on their incident response responsibilities.
+      description: |-
+        Personnel are knowledgeable about their role and responsibilities in incident response and
+        are able to access assistance and guidance when required.
+      levels:
+        - base
+      status: pending
+      controls:
+      - id: 12.10.4.1
+        title: The frequency of periodic training for incident response personnel is defined in
+          the entity's targeted risk analysis, which is performed according to all elements
+          specified in Requirement 12.3.1.
+        description: |-
+          Incident response personnel are trained at a frequency that addresses the entity's risk.
+          This requirement is a best practice until 31 March 2025, after which it will be required
+          and must be fully considered during a PCI DSS assessment.
+        levels:
+          - base
+        status: pending
+
+    - id: 12.10.5
+      title: The security incident response plan includes monitoring and responding to alerts from
+        security monitoring systems.
+      levels:
+        - base
+      status: pending
+
+    - id: 12.10.6
+      title: The security incident response plan is modified and evolved according to lessons
+        learned and to incorporate industry developments.
+      description: |-
+        The effectiveness and accuracy of the incident response plan is reviewed and updated after
+        each invocation.
+      levels:
+        - base
+      status: pending
+
+    - id: 12.10.7
+      title: Incident response procedures are in place, to be initiated upon the detection of
+        stored PAN anywhere it is not expected.
+      levels:
+        - base
+      status: pending
+
+  - id: A1.1
+    title: Multi-tenant service providers protect and separate all customer environments and data.
+    levels:
+      - base
```

Comment:
The OCP4 control file for PCI-DSS 3.2.1 had `A1.1` to `A1.4` on level `shared_hosting_provider`.
Do we see value in these levels?

Reply:
Not that I'm aware of.